at-luantran/Week4/Human_Resources/main.rb

```diff
@@ -0,0 +1,191 @@
+    p '3. Forgot Password'
+    p '4. Exit'
+    p 'Choose number:'
+    x = gets.chomp.to_i
+    case x
+    when 1
+      @user = SessionController.new.login
+      if @user
+        system('clear')
+        p "WELCOME #{@user.username}"
+        screen2
+      else
+        p 'Invalid username or password!!!'
+      end
+    when 2
+      SessionController.new.register
+    when 3
+      SessionController.new.forgot_password
+    when 4
+      exit!
+    else
+      system('clear')
+      p '=====>Notice! :Choose number 1-> 4'
+    end
+  end
+end
+
+def screen2
+  loop do
+    p '============ Human Resource Management ============='
+    p '1. Team management'
+    p '2. Leave management'
+    p '3. Notification management'
+    p '4. Logout'
+    p '5. Shutdown program'
+    p '6.CSV'
+    p 'Choose number:'
+    x = gets.chomp.to_i
+    case x
+    when 1
+      screen_team
+    when 2
+      screen_leave
+    when 3
+      screen_notif
+    when 4
+      p 'Logout'
+      @user = nil
+      screen1
+    when 5
+      exit!
+    else
+      p '=====>Notice! :Choose number 1-> 6'
+    end
+  end
+end
+
+def screen_team
+  system('clear')
+  loop do
+    p '========== Team Management ==========='
+    p '1. Create a new team(only root)' if @user.role == 'root'
+    p '2. Add a new member(only team lead)' if @user.role == 'team_lead'
+    p '3. Show list of members'
+    p '4. Return: Go to Level 1'
+    p '5. Logout: Go to Level 0'
+    p '6. Exit: Shutdown program'
+    p 'Choose number:'
+    x = gets.chomp.to_i
+    case x
+    when 1
+      TeamController.new
+      system('clear')
+      p 'Create team success!'
+    when 2
+      TeamController.add_member(@user)
+      system('clear')
+      p 'Add member success'
+    when 3
+      TeamController.members(@user)
+    when 5
+      p 'Logout'
+      @user = nil
+      screen1
+    when 6
+      exit!
+    else
+      p '=====>Notice! :Choose number 1-> 6'
+      break if x == 4
+    end
+  end
+end
+
+def screen_notif
+  system('clear')
+  loop do
+    p '========== Notification Management ==========='
+    p '1. Send a notification(only team lead)' if @user.role == 'team_lead'
+    p '2. Read a notification'
+    p '3. Delete a notification'
+    p '4. Show list of notifications.'
+    p '5. Return: Go to Level 1'
+    p '6. Logout: Go to Level 0'
+    p '7. Exit: Shutdown program'
+    p 'Choose number:'
+    x = gets.chomp.to_i
+    case x
+    when 1
+      NotificationController.send(@user)
+      system('clear')
+      p 'Send notification success!'
+    when 2
+      NotificationController.read(@user)
+    when 3
+      NotificationController.delete(@user)
+      system('clear')
+      p 'Delete notification success!'
+    when 4
+      NotificationController.read(@user)
+    when 6
+      p 'Logout'
+      @user = nil
+      screen1
+    when 7
+      exit!
+    else
+      p '=====>Notice! :Choose number 1-> 6'
+      break if x == 5
+    end
+  end
+end
+
+def screen_leave
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/b169340e04d51ef2ed0d2bc2bb6bf14408869e9a/at-luantran/Week4/Human_Resources/main.rb#L144"></span>Metrics/CyclomaticComplexity: Cyclomatic complexity for screen_leave is too high. [12/10]</td>
    </tr>
  </tbody>
</table>